Changelog.md

```diff
@@ -4,8 +4,8 @@ This file only reflects the changes that are made in this image. Please refer to
 
 **16.10.7**
 
-- gitlab: upgrade CE to v16.10.6
-- gitaly: upgrade to v16.10.6
+- gitlab: upgrade CE to v16.10.7
+- gitaly: upgrade to v16.10.7
 - gitlab-pages: upgrade to v16.10.6
```

Comment:
Please update this item to `gitlab-pages: upgrade to v16.10.7` (see https://github.com/sameersbn/docker-gitlab/blob/cc3ab205758df3f214b9d8b35da4e3dd11d256e2/Dockerfile#L10).